Scripting/Bindings.cs

```diff
@@ -256,11 +455,92 @@ public override void _Process(float Delta)
 					Scripting.ConsoleEngine.Execute($"{Binding.Function}(1)", Scripting.ConsoleScope);
 				}
 			}
+			else if(Binding.Type == TYPE.CONTROLLERAXIS)
+			{
+				
+				int VerticalAxis = 0; 
+				int HorizontalAxis = 0;
+				InputEventJoypadMotion StickEvent = null; 
+				
+
+				foreach(InputEvent Option in InputMap.GetActionList(Binding.Name)) {
+					if (Option is InputEventJoypadMotion JoyEvent) {
+						StickEvent = JoyEvent;
+					}
+				}
+				
+
+				if (StickEvent.Axis == 0 || StickEvent.Axis == 1)
+				{
+					// We are using Left stick to look around
+					VerticalAxis = 1;
+					HorizontalAxis = 0;
+				}
+				else if (StickEvent.Axis == 2 || StickEvent.Axis == 3)
+				{
+					// We are using Right stick to look around
+					VerticalAxis = 3;
+					HorizontalAxis = 2;
+				}
+				else
+				{
+					// Something has completely glitched
+				}
+				
+				if (Math.Abs(Input.GetJoyAxis(0,HorizontalAxis)) >= 0.25 || Math.Abs(Input.GetJoyAxis(0,VerticalAxis)) >= 0.25) 
```

Comment:
Can we make the deadzone value a float value in Game.cs? (I really need to create a dedicated class for config values)